app/lib/server/functions/createDirectRoom.js

```diff
@@ -1,45 +1,81 @@
 import { Rooms, Subscriptions } from '../../../models/server';
+import { getDefaultSubscriptionPref } from '../../../utils';
+import { callbacks } from '../../../callbacks';
```

Comment:
this way editor's ctrl+click works =)
```suggestion
import { getDefaultSubscriptionPref } from '../../../utils/server';
import { callbacks } from '../../../callbacks/server';
```